ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionValidatorTest.java

```diff
@@ -85,7 +95,7 @@ public void transactionWithInvalidChainIdShouldReturnWrongChainId() {
     validator = new PrivateTransactionValidator(Optional.empty());
 
     ValidationResult<TransactionInvalidReason> validationResult =
-        validator.validate(privateTransactionWithChainId(999), 0L);
+        validator.validate(privateTransactionWithChainId(Optional.of(BigInteger.ONE)), 0L);
```

Comment:
function is badly named I assumed this was:
transactionWithChainIdCannotBeUsedWithValidatorWithoutChainId?